binaries/scheduler/main.go

```diff
@@ -21,9 +23,9 @@ func main() {
 	parser.Workers[""] = &config.RPCWorkersConfig{Type: "rpc"}
 	parser.Cluster[""] = &local.ClusterLocalConfig{Type: "local"}
 	parser.Cluster["local"] = &local.ClusterLocalConfig{}
+	parser.Report[""].(*config.DefaultReportConfig).HttpAddr = fmt.Sprintf("localhost:%d", *httpPort)
 
 	// Construct scootapi server handler based on config.
-
 	handler, err := parser.Create([]byte(*cfgText))
```

Comment:
Let's keep this in mind and address it later when we're adding config to the other binaries.